get-started/hallucination-agent/chat-completion.md

```diff
@@ -0,0 +1,124 @@
+---
+title: Chat Completion Hallucination Detection
+description: Learn how to use the kluster.ai Hallucination Detection Agent to validate responses in full chat conversations.
+---
+
+# Chat Completion
+
+The Chat Completion method allows you to validate responses in full conversation histories using the same format as the standard chat completions API. This approach enables detection of hallucinations within the complete context of a conversation.
+
+## Prerequisites
+
+Before getting started with hallucination detection, ensure the following requirements are met:
+
+- **kluster.ai account** - sign up on the [kluster.ai platform](https://platform.kluster.ai/signup){target=_blank} if you do not have one
+- **kluster.ai API key** - after signing in, go to the [API Keys](https://platform.kluster.ai/apikeys){target=_blank} section and create a new key. For detailed instructions, see the [Get an API key](https://docs.kluster.ai/get-started/get-api-key/){target=_blank} guide
+
+## How it works
+
+The service evaluates the truthfulness of responses within a conversation by:
+
+1. Analyzing the entire conversation history, including system instructions.
+2. Examining the assistant's responses within context.
+3. Determining if the responses contain hallucinated or unsupported information.
+4. Providing a detailed explanation of the reasoning behind the determination and the search results used for fact-checking.
```

Comment:
I would put this in the introduction! :) and then at the end of the intro I would put something like:

"This page shows you how you can perform hallucination detection using the chat completions endpoint via the kluster.ai API."